docs/reference/upgrade.asciidoc

```diff
@@ -5,12 +5,12 @@
 --
 ifeval::["{release-state}"!="released"]
 [[upgrade-pre-release]]
-IMPORTANT: This documentation is for a pre-release of {es} {minor-version}. 
+IMPORTANT: This documentation is for a pre-release of {es} {minor-version}.
```

Comment:
I've got changes to this topic in flight. Revert these & I'll add the compatibility info where needed.  (And link to the new topic.)

Reply:
The whitespace changes were accidental and have been removed from the PR. Did you mean just the white space or the entire file ?